wfe/wfe.go

```diff
@@ -1333,7 +1340,22 @@ func (wfe *WebFrontEndImpl) Certificate(ctx context.Context, logEvent *requestEv
 
 	// TODO Content negotiation
 	response.Header().Set("Content-Type", "application/pkix-cert")
-	response.Header().Add("Link", link(issuerPath, "up"))
+	if features.Enabled(features.UseAIAIssuerURL) {
+		parsedCertificate, err := x509.ParseCertificate([]byte(cert.DER))
+		if err != nil {
+			logEvent.AddError("unable to parse certificate: %s", err)
+			wfe.sendError(response, logEvent, probs.Malformed("Unable to parse certificate"), err)
```

Comment:
This should be a `ServerInternal` since it's a certificate we have stored which should be valid and parsable.